spark-bigquery-pushdown/pushdown_common_src/test/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverterSuite.scala

```diff
@@ -318,6 +318,104 @@ class SparkExpressionConverterSuite extends AnyFunSuite with BeforeAndAfter {
+    val stringTranslateExpression = StringTranslate.apply(schoolIdAttributeReference, Literal("*"), Literal("**"))
+    val bigQuerySQLStatement = converter.convertStringExpressions(stringTranslateExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "TRANSLATE ( SUBQUERY_2.SCHOOLID , '*' , '**' )")
+  }
+
+  test("convertStringExpressions with StringTrim") {
+    val stringTrimExpression = StringTrim.apply(schoolIdAttributeReference)
+    val bigQuerySQLStatement = converter.convertStringExpressions(stringTrimExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "TRIM ( SUBQUERY_2.SCHOOLID )")
+  }
+
+  test("convertStringExpressions with StringTrimLeft") {
+    val stringTrimLeftExpression = StringTrimLeft.apply(schoolIdAttributeReference)
+    val bigQuerySQLStatement = converter.convertStringExpressions(stringTrimLeftExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "LTRIM ( SUBQUERY_2.SCHOOLID )")
+  }
+
+  test("convertStringExpressions with StringTrimRight") {
+    val stringTrimRightExpression = StringTrimRight.apply(schoolIdAttributeReference)
+    val bigQuerySQLStatement = converter.convertStringExpressions(stringTrimRightExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "RTRIM ( SUBQUERY_2.SCHOOLID )")
+  }
+
+  test("convertStringExpressions with Upper") {
+    val upperExpression = Upper.apply(schoolIdAttributeReference)
+    val bigQuerySQLStatement = converter.convertStringExpressions(upperExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "UPPER ( SUBQUERY_2.SCHOOLID )")
+  }
+
+  test("convertStringExpressions with StringInstr") {
+    val stringInstrExpression = StringInstr.apply(schoolIdAttributeReference, Literal("1234"))
+    val bigQuerySQLStatement = converter.convertStringExpressions(stringInstrExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "INSTR ( SUBQUERY_2.SCHOOLID , '1234' )")
+  }
+
+  test("convertStringExpressions with InitCap") {
+    val initCapExpression = InitCap.apply(schoolIdAttributeReference)
+    val bigQuerySQLStatement = converter.convertStringExpressions(initCapExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "INITCAP ( SUBQUERY_2.SCHOOLID )")
+  }
+
+  test("convertStringExpressions with RegExpExtract") {
+    val regExpExtractExpression = RegExpExtract.apply(schoolIdAttributeReference, Literal("[0-9]"), Literal(1))
+    val bigQuerySQLStatement = converter.convertStringExpressions(regExpExtractExpression, fields)
+    assert(bigQuerySQLStatement.isDefined)
+    assert(bigQuerySQLStatement.get.toString == "REGEXP_EXTRACT ( SUBQUERY_2.SCHOOLID , '[0-9]' , 1 )")
```

Comment:
According to the [doc](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#regexp_extract) it should be `REGEXP_EXTRACT ( SUBQUERY_2.SCHOOLID , r'[0-9]' , 1 )`

Reply:
Addressed